website/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+	"compilerOptions": {
+		"target": "es5",
```

Comment:
@andrashee Would probably go to at least es6 or better something like es2018 or es2020.